
public class Sec33_290_ExceptionsDemo {
	int a=4;
	int b=7;
	int c=0;
	
	try {
	int k=b/c;
	} catch{
		
		
	}
	
	public void getData() {
		System.out.println(k);
	}

	public static void main(String[] args) {
		// TODO Auto-generated method stub

	}

}
